pyvista/utilities/helpers.py

```diff
@@ -1458,3 +1459,56 @@ def cubemap(path='', prefix='', ext='.jpg'):
         texture.SetInputDataObject(i, flip.GetOutput())
 
     return texture
+
+
+def set_default_active_vectors(mesh: 'pyvista.DataSet') -> None:
+    """Set a default vectors on mesh, if not already set.
```

Comment:
I know you mean "vectors" as an abstract entity, but the singular-plural combo reads surprisingly. I would either use "set default vectors" or "set default vectors data" or "set a default vectors array". There's a similar instance in the third paragraph but that's less critical as it isn't in the header line.

Reply:
I took your suggestion.